src/status_im/subs/wallet/wallet.cljs

```diff
@@ -411,6 +411,7 @@
  (fn [[account networks] [_ query chain-ids]]
    (let [tokens        (map (fn [token]
                               (assoc token
+                                     :bridge-disabled?  (not (send-utils/bridgeable? (:symbol token)))
```

Comment:
I would suggest to use either of the namings `bridge-disabled?` or `bridgeable?` as they mean the same thing